Cargo.toml

```diff
@@ -29,3 +32,4 @@ object = { version = "0.32", default-features = false, features = [
     "std",
 ] }
 indicatif = "0.17.7"
+serialport = { version = "4.2.2", default-features = false}
```

Comment:
Test on Linux. This disables libudev linking, thus serialport can't acquire USB dev info.